server/PokerTableObjects/PokerTable.js

```diff
@@ -0,0 +1,558 @@
+let PokerPlayerSeat = require("./PokerPlayerSeat.js");
+
+// this is a pokerTable class which will handle the current poker table state
+// it contains info on the players, community cards, bets, etc.
+module.exports = class PokerTable {
+    constructor(bigBlind) {
+        this.numberOfTableSeats = 6;
```

Comment:
This variable seems to be common between multiple files as well (declared also in ChipDistributionCalculator - line 4)

Reply:
added PokerUtilities.js file to hold this value